inventories/inventory_aws/group_vars/all.yml

```diff
@@ -0,0 +1,43 @@
+cloud: "aws"
+region: "eu-central-1"
+# aws.networking
+vpc_name:       'Yugabyte_VPC_Res'
+vpc_cidr_2:     '185'
+vpc_cidr: '10.{{vpc_cidr_2}}.0.0/16'
+
+subnets:
+  - type: public
+    cidr: "10.{{vpc_cidr_2}}.10.0/24"
+    az: a
+  - type: public
+    cidr: "10.{{vpc_cidr_2}}.11.0/24"
+    az: b
+  - type: public
+    cidr: "10.{{vpc_cidr_2}}.21.0/24"
+    az: c
+
+route_name: "yugabyte_route"
+instances_per_subnet: 2
+
+# Internet Gateway Name
+igw_name: "Yugabyte_igw"
+
+ec2_tag: "Yugabyte_nodes"
+ansible_user: "centos"
+device_name: /dev/xvda
+vol_type: gp2
+vol_size: 8
+
+#The local path to which we would save our EC2 Private Key
+#if the keypair name alreay exists within amazon, it will pick the location key, otherwise it stores it in that directory
+ec2_key_directory: "YOUR KEY PATH"
+keypair_name: "ec2_key_pair"
+
+instance_product_code: "aw0evgkw8e5c1q413zgy5pjce"
+SGDBName: "Yugabyte_cluster" #Security db group name
+SGDBInfra: "infra_secgroup" #Security infra group name
```

Comment:
why are these two variables named differently. why can't we call it `yugabyte-db-sg` and `yugabyte-infra-sg`

Reply:
Oh, I thought to name them differently just to seggregate a little bit db specific ports and infrastructure related ones. I can change that variable name. No problem.

Reply:
Some Ansible did not recognize `yugabyte-db-sg` and `yugabyte-infra-sg` variables and was only recognizing yugabyte substring without `-db-*` part. Therefore I created: `yugabyte_db_sg` and `yugabyte_infra_sg` variables.